lesson02/assignment/css/_components.content.scss

```diff
@@ -1,17 +1,84 @@
-html {
-    box-sizing: border-box;
+/* media queries to make responsive for different screens */
+@media screen and (max-width: 576px) {
+        .main {
+            grid-template-columns: 1fr;
+            margin: .5rem;
+            padding: .5rem;
+            border: solid $gray 1rem;
+        }
+        .box2 {
+          display: grid;
+          grid-column-start: 1;
+          grid-column-end: 2;
+        }
 }
-
-*,
-*:after,
-*:before {
-    box-sizing: inherit;
+@media screen and (min-width: 576px) {
+        .main {
+            grid-template-columns: 1fr 2fr;
+            margin: 40px;
+            padding: 30px;
+            border: solid $gray 40px;
+        }
+        .box2 {
+            display: grid;
+            grid-column-start: 2;
+            grid-column-end: 3;
+        }
 }
-
+/* CSS styling total page */
 .main {
-    padding: 2rem;
+  font: $font-fam;
+  display: grid;
+  grid-gap: 0px 5px;
```

Comment:
This is an okay usage of px as a unit of measurement. They're also totally fine when defining borders.